cmake/Modules/FindPythia6.cmake

```diff
@@ -4,9 +4,17 @@
 # Defines:
 #  PYTHIA6_FOUND
 #  PYTHIA6_LIBRARIES
+set(TEST_PYTHIA6_ROOT_DIR  "" ${PYTHIA6_ROOT_DIR})
+IF(TEST_PYTHIA6_ROOT_DIR STREQUAL "")
+  IF(DEFINED ENV{PYTHIA6_ROOT_DIR})
+    set(PYTHIA6_ROOT_DIR  $ENV{PYTHIA6_ROOT_DIR})
+  else()
+    set(PYTHIA6_ROOT_DIR  "/usr")
+  endif()
+endif()
 
-find_library(PYTHIA6_LIBRARY NAMES Pythia6 pythia6-$ENV{PYTHIA6_VERSION}
-			     HINTS $ENV{PYTHIA6} $ENV{PYTHIA6}/lib)
+find_library(PYTHIA6_LIBRARY NAMES Pythia6 pythia6 pythia6-$ENV{PYTHIA6_VERSION}
+			     HINTS $ENV{PYTHIA6} $ENV{PYTHIA6}/lib $ENV{PYTHIA6}/lib64 ${PYTHIA8_ROOT_DIR}/lib ${PYTHIA8_ROOT_DIR}/lib64)

```

Comment:
Should that be `PYTHIA6` rather than `PYTHIA8` in the last two variables?

Reply:
Hi @drbenmorgan, that was a very first PR to lay a fundament to switch examples from hepMC2 to HEpMC3.
Practically I can continue if  you have some time.

Reply:
Thanks @andriish, this all looks good now so I'll patch it in upstream and close things here when it's merged.

I do have some time to look at the HepMC2->3 switch (I guess that's the changes in #18?) so if you want to revive/rebase that, I can take a look.

Reply:
Hi @drbenmorgan,

- CI -- see #67. That is quite a solid  and fast solution. 
- cmake modules for Pythia6/8/LHAPDF can be added within minutes, HepMC3 provides a config for `cmake`, so no module is needed. But this makes sense to do after the CI.
-  The examples themselves should come either togather with cmake modules or later, separately.

Amdrii